editor/src/clj/editor/recent_files.clj

```diff
@@ -25,7 +25,7 @@
             [editor.ui :as ui]
             [editor.ui.fuzzy-choices :as fuzzy-choices]
             [editor.workspace :as workspace])
-  (:import [javafx.scene.control TabPane]))
+  (:import [javafx.scene.control SplitPane Tab TabPane]))
```

Comment:
Let's add `(set! *warn-on-reflection* true)` after the `ns` declaration and ensure there are no reflection warnings.